code/src/components/Labyrinth.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import StartScreen from './StartScreen'
+import GameScreen from './GameScreen'
+import { Loading } from './Loading'
+
+import styled from 'styled-components'
+
+import {
+  imgURL_start, 
+  imgURL_1_1, 
+  imgURL_1_3,
+  imgURL_1_0,
+  imgURL_0_3,
+  imgURL_0_2,
+  imgURL_0_1,
+  imgURL_0_0
+} from '../images/images'
+
+const Main = styled.main`
+  display: flex;
+  color: white;
+  justify-content: center;
+  position: relative;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+  background-image: ${(props) =>
+    props.coordinates === '1,3'
+      ? imgURL_1_3
+      : props.coordinates === '1,1'
+      ? imgURL_1_1
+      : props.coordinates === '1,0'
+      ? imgURL_1_0
+      : props.coordinates === '0,3'
+      ? imgURL_0_3
+      : props.coordinates === '0,2'
+      ? imgURL_0_2
+      : props.coordinates === '0,1'
+      ? imgURL_0_1
+      : props.coordinates === '0,0'
+      ? imgURL_0_0
+      : imgURL_start};
```

Comment:
if you name them `imgURL-1,3` then you can skip all the logic with `imgURL-${props.coordinates}` 🤓